docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+draw in visitors for your main website and can keep your audience engaged. Of
+course, you can blog about any topics close to your heart.
+
+The [blog plugin] makes running a blog alongside your other content easy but you
+can also configure it to run a stand-alone blog if posts are the only kind
+of content you need. It becomes even more powerful when used in combination with
+other features of Material for MkDocs such as the [tags plugin] or with third
+party plugins such as the [RSS plugin].
+
+[tags plugin]: ../../plugins/tags
+[RSS plugin]: https://guts.github.io/mkdocs-rss-plugin/
+
+!!! example "Explore the Material for MkDocs Blog"
+
+    Take a moment to look at the [Material for MkDocs blog] to see what elements
+    a blog consists of before moving on to the next section. What makes the blog
+    different from the rest of the site?
+
+[Material for MkDocs blog]: https://squidfunk.github.io/mkdocs-material/blog/
+
+## 3. Key concepts
+
+A blog consists of a number of self-contained _posts_ (also often called
+articles).
+
+An index page shows the posts in reverse chronological order, with the most
+recent post at the top. It usually shows only a short _excerpt_ and a link that
+the user can click to navigate to the full post.
+
+Both the index page and the post itself usually list information such as
+when you published the post, when you updated it, who the author is and what the
+expected reading time is.
+
+Since the blog posts are primarily arranged by time and not into a hierarchy,
+their URLs do not reflect such a structure. Instead, each post's URL usually
+contains a shortened description, the _slug_, which is usually derived from
+the first heading in the post.
+
+The main navigation structure is the timeline, which you can subdivide into
+_categories_. In addition, posts can be _tagged_ to provide an additional
+navigation structure based on content.
+
+The navigation elements in the blog are the timeline, with the main index page
+showing a given number of posts and an _archive_ section allowing access to
+older posts, organized by year. The _categories_ section provides access to
+index pages for the categories. In addition, when using tagging, you can also
+create tag index pages.
+
+Navigation by _author_ helps users to find posts by a specific user when there
+are more than one. Each author index page lists posts by that author in reverse
+chronological order and can provide more information about them.
+
+Finally, an _RSS feed_ allows users to subscribe to a blog so that they get
+notified when you publish new posts. RSS Feed readers are often used to access
+blogs that a user follows. They usually support downloading the blog content for
+offline consumption.
+
+## 4. Setting up your blog
+
+The blog plugin is part of Material for MkDocs but you need to configure it
+in the `mkdocs.yml`.
+
+!!! example "Set up a blog"
+
+    If you have not done so already, create a project for your blog:
+
+    === "MacOS/Linux"
```

Comment:
Those are prerequisites. Please don't add OS specific instructions unless absolutely necessary. It was mentioned above that the user already worked through the creating your site guide, which is exactly this.

Reply:
Right, when I wrote that people should have worked through the getting started guide I was thinking about what they know, not what they have on the disks. Good point and this does simplify things.